packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -502,14 +502,18 @@ export class AccountResolver {
     @GqlXHeaders() headers: Headers,
     @Args('input', { type: () => AccountResetInput })
     input: AccountResetInput
-  ) {
-    return this.authAPI.accountReset(
+  ): Promise<AccountResetPayload> {
+    const result = await this.authAPI.accountReset(
```

Comment:
I'm not 100% certain this was necessary, but I changed this while debugging and it now matches other resolver functions result/returns. We already have tests written for it.